career/elevator_pitch.md

```diff
@@ -0,0 +1,68 @@
+Whenever you are in a position where you quickly need to tell someone about yourself with limited time, then the elevator pitch is an excellent tool.
+
+The following paragraph (shamelessly taken from wikipedia) sums it up quite nicely;
+
+> An elevator pitch, elevator speech, or elevator statement is a short
+> description of an idea, product, company, or oneself that explains the
+> concept in a way such that any listener can understand it in a short
+> period of time.
+
+This description typically explains who the person is, what he/she does, and if he or she has the relevant experience. For the latter, when explaining an individual, the description generally explains one’s skills and goals, and why they would be a productive and beneficial person to have on a team or within a company/project.
+
+An elevator pitch does not have to include all of these components. But it usually has at least some explanations of what the idea, product, company, or person is and why it/they are valuable.
+
+
+Your assignment is to create a elevator pitch for when you are networking and meeting recruiters. The pitch should sound natural and have an easy flow. The pitch should not sound mechanical or rehearsed, even though you will rehearse it over and over again.
```

Comment:
`Your assignment is to create a elevator ` a should be an

Reply:
Done 👍